core/src/test/scala/com/mesosphere/usi/core/AkkaUnitTest.scala

```diff
@@ -0,0 +1,14 @@
+package com.mesosphere.usi.core
+
+import akka.actor.ActorSystem
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{BeforeAndAfterAll, Suite}
+
+trait AkkaUnitTest extends Suite with BeforeAndAfterAll with ScalaFutures {
```

Comment:
See `test-utils` from @zen-dog 🙂

Reply:
how is that related? since this is unit test? 🤔

Reply:
Alex imported `AkkaUnitTest` as well. `test-utils` is not just for integration tests.